### ISSUE TITLE
Celtic Training Bug

### ISSUE THREAD

**Issue by xirsoi** [xirsoi/ancientreligions#42]
The Celtic training/coming of age events are flavorful and neat, but need some tweaking.
One of the events, where your child comes home, has a description of "Child returns home." Clearly a placeholder, but we should probably put a little more prose into it ;)

Additionally, the child shouldn't return from their training until right before or right after they reach adulthood. Currently they seem to come back a few months later and it seems weird. I sent my kid to train with the druids, 4 months later (still 14 years old!) they come back a full fledged Druid, trait and all?

Something that came up in my research was that children began their education at the age of 7 (boys and girls) and continued until 18 for boys and 14 for girls. The girls education could be extended to 18 by their parents if they pleased. In game, this would probably be a monetary and/or piety cost, with the benefit of increasing the likelihood the child will get good education traits.

**Comment by xirsoi** [comment]
After examining the events I've determined that most of them were written with the intent that they would be used with an event where an adult son asks the liege-parent if they can become a druid. Its possible this was one of the areas that super had been overhauling.

I've added the condition that the return events cannot happen before the age of 18. I've also done my best to make the events open to females as well, since Celtic didn't discriminate. I'll be committing soon.
